### ISSUE TITLE
Extension development window is not torn down when debugging stops

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#124128]
1. Debug an extension
2. Hit stop
3. The extension host window is still alive

I'm pretty sure this is new today, I was doing js-debug work yesterday and didn't notice it (but might have been behind an Insiders)

**Comment by connor4312** [comment]
Interestingly, it seems to work on a hello world extension, but not in js-debug ([config](https://github.com/microsoft/vscode-js-debug/blob/8f4e2489916e61c9ce7b4a561dfaefdde80d1795/.vscode/launch.json#L55-L69)). In the logs I see the DAP sessions being torn down gracefully.

**Comment by alefragnani** [comment]
I'm facing the same issue on current **Insiders** and **Stable**, while debugging any my VS Code extensions.

**Comment by isidorn** [comment]
Sorry for the slow response here, looks like a duplicate of https://github.com/microsoft/vscode/issues/126211
Closing and if you can repro with latest insiders let me know and I can reopen

**Comment by alefragnani** [comment]
It is still happening on Insiders, at least on my scenario (debugging VS Code extensions)

```
Version: 1.58.0-insider
Commit: bfac07cd0e374c433c2c06d6999dabf9cf9d5d29
Date: 2021-06-28T05:14:49.021Z
Electron: 12.0.12
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 20.5.0
```

**Comment by isidorn** [comment]
Ok let me reopen and I can investigate more

**Comment by isidorn** [comment]
I tried to reproduce and I could not. Here are my steps.

1. Open `js-debug` repo, choose Extension launch config
2. Press F5 > new window opens
3. click on the toolbar Stop action -> second window closes as expected

I tried this 10+ times and every time it worked.
Are you doing something different.
Please note that I have pushed a potential fix for this today, so please try out of source or with Insiders tomorrow.

**Comment by alefragnani** [comment]
I've tested with some of my own extensions, which seems to be simpler than `js-debug`, so I'm not pretty sure what should be the cause. I've tested with two extensions from `vscode-extension-samples` with no problem.

The only thing I can easily see as differences between my own extensions and those from `vscode-extension-samples` and `js-debug` extensions is the fact that my extension uses **Webpack**.

I'll test it again tomorrow, and let you know the results.

Thank you
